todo-app/src/todoList.js

```diff
@@ -0,0 +1,15 @@
+let todosList = [
```

Comment:
You can remove this `todo-app/src/todoList.js` file as it is not used anymore, right?